swift-sdk/IterableAPI.swift

```diff
@@ -114,6 +114,24 @@ import UIKit
         internalImplementation?.removeDeviceAttribute(name: name)
     }
     
+    /// Set the user of the SDK instance to the email address specified
+    ///
+    /// - parameter email: the email of the user for the SDK instance
+    /// - parameter token: the associated authentication token for the user
+    @objc(setEmail:token:)
+    public static func setEmail(_ email: String, _ token: String? = nil) {
```

Comment:
Recommendation: rename to `setEmail(_ email: String, withToken token: String? = nil)`

Reply:
Also objc name to `setEmail:withToken`